pegasus/src/env.rb

```diff
@@ -49,11 +53,15 @@ def load_pegasus_settings
   I18n.backend = CDO.i18n_backend
   I18n.backend.class.send(:include, I18n::Backend::Fallbacks)
   I18n.fallbacks = I18n::Locale::Fallbacks.new(['en-US'])
-  if rack_env?(:development) && !CDO.load_locales
+  if (rack_env?(:development) || rack_env?(:test)) && !CDO.load_locales
```

Comment:
Loading all HoC translations in a test environment is slow, which then cause unrelated Eyes tests to fail. (SauceLabs has a 60s timeout for each command.)